h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
No, I'm afraid this is not a good idea. If I see a file in `vendor/`, I will assume it is an unpatched version of an upstream project. If I subsequently update it with a patch release from upstream, these changes will be lost and the code that relies on this behaviour will break.

I'd suggest you look into a way of achieving your aims without patching files in `vendor/`.